getLocalStorage(key).then(value) TypeScript error

**Describe the bug**
```ts
/// <reference types="cypress-localstorage-commands" />

export const getLocalStorage = (key: string): any =>
  cy.getLocalStorage(key).then(value => {
    if (value === null) {
      return value;
    } else {
      return JSON.parse(value);
    }
  });
```
```
error TS2345: Argument of type 'Element' is not assignable to parameter of type 'string'.
```

**Expected behavior**
`value` should be `string | null`.

**Logs**
If applicable, add logs to help explain your problem.

**Operating system, Node.js an npm versions, or browser version (please complete the following information):**
 - OS: macOS 10.15.4
 - Node.js: 12.16.3
 - npm: 6.14.4

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 ### BREAKING CHANGES
 
+## [1.2.1] - 2020-06-09
+
+### Fixed
+- fix(TypeScript): Fix TypeScript return types declarations
+
+### Changed
+- chore(deps): Update devDependencies
+
 ## [1.2.0] - 2020-05-31
 
 ### Added

--- a/index.d.ts
+++ b/index.d.ts
@@ -6,26 +6,26 @@ declare namespace Cypress {
      * Command to save current localStorage values into an internal "snapshot"
      * @example cy.saveLocalStorage()
     */
-    saveLocalStorage(): Chainable<Element>
+    saveLocalStorage(): Chainable<undefined>
 
     /**
      * Command to restore localStorage to previously "snapshot" saved values
      * @example cy.restoreLocalStorage()
     */
-    restoreLocalStorage(): Chainable<Element>
+    restoreLocalStorage(): Chainable<undefined>
 
     /**
      * Command to clear localStorage "snapshot" values
      * @example cy.clearLocalStorageSnapshot()
     */
-    clearLocalStorageSnapshot(): Chainable<Element>
+    clearLocalStorageSnapshot(): Chainable<undefined>
 
     /**
      * Command to get localStorage item value
      * @param {string} itemKeyName - localStorage item to get
      * @example cy.getLocalStorage("cookies-accepted")
     */
-    getLocalStorage(itemKeyName: string): Chainable<Element>
+    getLocalStorage(itemKeyName: string): Chainable<string|null>
 
     /**
      * Command to set localStorage item value
@@ -33,13 +33,13 @@ declare namespace Cypress {
      * @param {string} value - value to be set
      * @example cy.setLocalStorage("cookies-accepted", "true")
     */
-    setLocalStorage(itemKeyName: string, value: string): Chainable<Element>
+    setLocalStorage(itemKeyName: string, value: string): Chainable<undefined>
 
     /**
      * Command to remove localStorage item
      * @param {string} itemKeyName - localStorage item to remove
      * @example cy.removeLocalStorage("cookies-accepted")
     */
-    removeLocalStorage(itemKeyName: string): Chainable<Element>
+    removeLocalStorage(itemKeyName: string): Chainable<undefined>
   }
 }

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "eslint-plugin-react": "7.20.0",
     "husky": "4.2.5",
     "jest": "26.0.1",
-    "lint-staged": "10.2.6",
+    "lint-staged": "10.2.7",
     "prettier": "2.0.5",
     "sinon": "9.0.2"
   },

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "eslint-plugin-react": "7.20.0",
     "husky": "4.2.5",
     "jest": "26.0.1",
-    "lint-staged": "10.2.7",
+    "lint-staged": "10.2.9",
     "prettier": "2.0.5",
     "sinon": "9.0.2"
   },

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cypress-localstorage-commands",
-  "version": "1.2.0",
+  "version": "1.2.1",
   "description": "Extends Cypress' cy commands with localStorage methods. Allows preserving localStorage between tests",
   "keywords": [
     "cypress",
@@ -50,7 +50,7 @@
     "@stryker-mutator/jest-runner": "3.2.4",
     "babel-eslint": "10.1.0",
     "coveralls": "3.0.9",
-    "eslint": "7.1.0",
+    "eslint": "7.2.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-prettier": "3.1.3",
     "eslint-plugin-react": "7.20.0",

--- a/sonar-project.properties
+++ b/sonar-project.properties
@@ -1,6 +1,6 @@
 sonar.organization=javierbrea
 sonar.projectKey=cypress-localstorage-commands
-sonar.projectVersion=1.2.0
+sonar.projectVersion=1.2.1
 
 sonar.javascript.file.suffixes=.js
 sonar.sourceEncoding=UTF-8

--- a/test-e2e/react-app/package.json
+++ b/test-e2e/react-app/package.json
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@data-provider/core": "2.2.1",
+    "@data-provider/core": "2.4.0",
     "@data-provider/browser-storage": "2.0.4",
     "@data-provider/react": "1.0.3",
     "prop-types": "15.7.2",

--- a/test-e2e/react-app/package.json
+++ b/test-e2e/react-app/package.json
@@ -40,7 +40,7 @@
     "react-scripts": "3.4.1",
     "babel-plugin-module-resolver": "4.0.0",
     "cypress": "4.7.0",
-    "serve": "11.3.1",
+    "serve": "11.3.2",
     "start-server-and-test": "1.11.0"
   }
 }

--- a/test-e2e/react-app/package.json
+++ b/test-e2e/react-app/package.json
@@ -39,7 +39,7 @@
   "devDependencies": {
     "react-scripts": "3.4.1",
     "babel-plugin-module-resolver": "4.0.0",
-    "cypress": "4.7.0",
+    "cypress": "4.8.0",
     "serve": "11.3.2",
     "start-server-and-test": "1.11.0"
   }

--- a/test-e2e/react-app/package.json
+++ b/test-e2e/react-app/package.json
@@ -39,7 +39,7 @@
   "devDependencies": {
     "react-scripts": "3.4.1",
     "babel-plugin-module-resolver": "4.0.0",
-    "cypress": "4.6.0",
+    "cypress": "4.7.0",
     "serve": "11.3.1",
     "start-server-and-test": "1.11.0"
   }

--- a/test-e2e/typescript/cypress/integration/assertions-example.ts
+++ b/test-e2e/typescript/cypress/integration/assertions-example.ts
@@ -22,6 +22,9 @@ describe("localStorage cookies-accepted item", () => {
 
   it("should be true after reloading", () => {
     cy.getLocalStorage("cookies-accepted").then((cookiesAccepted) => {
+      if (cookiesAccepted !== null) {
+        expect(JSON.parse(cookiesAccepted)).to.equal(true);
+      }
       expect(cookiesAccepted).to.equal("true");
     });
   });

--- a/test-e2e/typescript/package.json
+++ b/test-e2e/typescript/package.json
@@ -16,8 +16,8 @@
     "test:ci": "npm run copy:library && npm run build-and-serve-and-cypress"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "3.1.0",
-    "@typescript-eslint/parser": "3.1.0",
+    "@typescript-eslint/eslint-plugin": "3.2.0",
+    "@typescript-eslint/parser": "3.2.0",
     "cypress": "4.7.0",
     "eslint": "7.1.0",
     "eslint-config-prettier": "6.11.0",

--- a/test-e2e/typescript/package.json
+++ b/test-e2e/typescript/package.json
@@ -18,8 +18,8 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "3.2.0",
     "@typescript-eslint/parser": "3.2.0",
-    "cypress": "4.7.0",
-    "eslint": "7.1.0",
+    "cypress": "4.8.0",
+    "eslint": "7.2.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-prettier": "3.1.3",
     "start-server-and-test": "1.11.0",

--- a/test-e2e/typescript/package.json
+++ b/test-e2e/typescript/package.json
@@ -16,8 +16,8 @@
     "test:ci": "npm run copy:library && npm run build-and-serve-and-cypress"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "3.0.2",
-    "@typescript-eslint/parser": "3.0.2",
+    "@typescript-eslint/eslint-plugin": "3.1.0",
+    "@typescript-eslint/parser": "3.1.0",
     "cypress": "4.7.0",
     "eslint": "7.1.0",
     "eslint-config-prettier": "6.11.0",

--- a/test-e2e/typescript/package.json
+++ b/test-e2e/typescript/package.json
@@ -23,6 +23,6 @@
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-prettier": "3.1.3",
     "start-server-and-test": "1.11.0",
-    "typescript": "3.9.3"
+    "typescript": "3.9.5"
   }
 }

